app/db.py

```diff
@@ -449,7 +449,7 @@ def insert_data_conversations(self, data):
                 last_value = 0
             if data.id is None:
                 last_value += 1
-            data.id = last_value
+                data.id = last_value
             session.add(data)
```

Comment:
This was the error. The lack of indentation tried to update everything as the last object in the database.